### PR TITLE
Add Javadoc to JSONWriter

### PR DESCRIPTION
**Pull request by dzelemba**
This Javadoc makes the usage of JSONWriter more clear without having to dig into source code to understand difference between write & writeAttributes and explaining the DicomInputHandler part.

### CODE REVIEW

File: dcm4che-json/src/main/java/org/dcm4che3/json/JSONWriter.java [hunk 1]
```diff
@@ -78,12 +108,20 @@ public JSONWriter(JsonGenerator gen) {
         this.gen = gen;
     }
 
+    /**
+     * Writes the given attributes as a full JSON object. Subsequent calls will generate new a JSON
```
Comment (hczedik) [inline dcm4che-json/src/main/java/org/dcm4che3/json/JSONWriter.java]: please correct word order: "... generate a new JSON object."

File: dcm4che-json/src/main/java/org/dcm4che3/json/JSONWriter.java [hunk 2]
```diff
@@ -78,12 +108,20 @@ public JSONWriter(JsonGenerator gen) {
...
+     * object.
+     */
     public void write(Attributes attrs) {
         gen.writeStartObject();
         writeAttributes(attrs);
         gen.writeEnd();
     }
 
+    /**
+     * Writes the given attributes to JSON. Can be used to ouput multiple attributes (e.g. metadata,
```
Comment (hczedik) [inline dcm4che-json/src/main/java/org/dcm4che3/json/JSONWriter.java]: typo: ouput -> output

**Comment by dzelemba** [comment]
Typos fixed, please take another look!

**Comment by hczedik** [comment]
Great, thank you @dzelemba !

**Pull request merged**